controllers/commentsControllers.js

```diff
@@ -41,5 +41,24 @@ function postComment(request, response, next) {
      });
  }
 
-module.exports = { getComments , postComment };
+ function deleteComment (request, response, next) {
+
+  const { comment_id } = request.params;
+  
+  checkCommentExists(comment_id) 
+  .then((commentExists) => {
+    if (!commentExists) {
+      return Promise.reject({ status: 404, msg: 'Not Found' });
+    }
+    return deleteCommentModel(comment_id)
+  }).then(() => {
+    response.status(204).send()
```

Comment:
`res.sendStatus()` is also a good alternative to use here, when there is no body being sent